gsuite-wrapper/docker/docker-compose.yml

```diff
@@ -22,11 +25,19 @@ services:
       NODE_ENV: "${MYAEGEE_ENV}"
       # BUGSNAG_KEY: "${BUGSNAG_KEY_GSUITE_WRAPPER}"
     labels:
-      - "traefik.backend=gsuite-wrapper"
-      - "traefik.port=8084"
-      - "traefik.hc.frontend.rule=Path:/api/gsuite-wrapper/healthcheck;PathPrefixStrip:/api/gsuite-wrapper"
-      - "traefik.doc.frontend.rule=Path:/api/gsuite-wrapper/api-docs.json;PathPrefixStrip:/api/gsuite-wrapper"
-      - "traefik.enable=true"
+      #- "traefik.hc.frontend.rule=Path:/api/gsuite-wrapper/healthcheck;PathPrefixStrip:/api/gsuite-wrapper"
+      #- "traefik.doc.frontend.rule=Path:/api/gsuite-wrapper/api-docs.json;PathPrefixStrip:/api/gsuite-wrapper"
+      # ---------old ^ for some reason it was all that was exposed, TODO: check
```

Comment:
Are you sure that's all what was exposed? I thought we used gsuite-wrapper on prod once, since it is deployed there

Reply:
well weirdly, yes, but again that's why i wrote i have to double check